.env.template

```diff
@@ -4,3 +4,8 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+
+# Trello keys 
+TRELLO_KEY
```

Comment:
you should add a little explanation in the README about what these environment variables are, and how to go about getting them.